app/src/main/res/layout/fragment_popup_attribute_list.xml

```diff
@@ -59,18 +61,70 @@
                     app:layout_constraintTop_toTopOf="parent"
                     app:layout_constraintVertical_bias="0.466" />
 
+            <ImageView
+                    android:id="@+id/edit"
+                    android:layout_width="30dp"
+                    android:layout_height="34dp"
```

Comment:
The sizes all seem to be hardcode here and in the related row xml. Have you tested this out on different form factors, or something like that?

Reply:
I have tested this out on a phone and a tablet. The UI still needs work though and I will be addressing it in the next issue.